dev/io.openliberty.org.apache.myfaces.4.0_fat/bnd.bnd

```diff
@@ -33,5 +34,5 @@ javac.target: 11
 	net.sourceforge.htmlunit:htmlunit;version='2.44.0',\
 	io.openliberty.org.apache.commons.codec;version=latest,\
 	io.openliberty.org.apache.xercesImpl;version=latest,\
-	xml-apis:xml-apis;version='1.4.01',\
```

Comment:
This was originally added to address the following issue: https://github.com/OpenLiberty/open-liberty/issues/22946 I just want to ensure we're not regressing anything.

Reply:
`org.w3c.dom.*` and `org.xml.sax.*` packages both exist in the `java.xml` module in Java 11 and should be available for our tests to use natively from the JDK. 

My guess is that the reason you were getting issues before was that this project was set up to be compiled at Java 11, but the classpath file didn't get updated for it to be module compatible. 
https://github.com/OpenLiberty/open-liberty/blob/c71e04594d78795a7e92632ef02d66055f30df29/dev/io.openliberty.org.apache.myfaces.4.0_fat/.classpath#L12-L14

Was there a certain Java level, JDK, or OS that having issues accessing the java.xml module? 
If so I can make sure my personal build tests it to make sure this won't cause a regression.

Reply:
The test failures summary is: `Test failure: io.openliberty.org.apache.myfaces.4.0_fat AlwaysPassesTest fails to produce a result on all temurin 8 platforms` The AlwaysPassesTest I believe is something created by the build infrastructure. I send you a slack message as well to discuss further.

Reply:
Thanks for the feedback.  I believe this could work without the extra xml bundle, but I'll leave that to future investigation.  It looks like the builds that failed were when running using Java 8 with the Temurin JDK.  However, this test suite is only supposed to run on Java 11 JDKs so I'm thinking this must be something quirky with our new CI builds.